src/main/java/com/conveyal/datatools/common/status/MonitorableJob.java

```diff
@@ -36,8 +36,8 @@
      */
     public boolean active = false;
 
-    protected String parentJobId;
-    protected JobType parentJobType;
+    public String parentJobId;
+    public JobType parentJobType;
```

Comment:
These might deserve some comments to indicate that they are needed in the JSON response.

Reply:
Updated in https://github.com/ibi-group/datatools-server/pull/406/commits/5c99775627b6b504811a87fb2c548ee2e948cf2e.